internal/cli/root.go

```diff
@@ -93,6 +95,15 @@ func (c *Client) apiConnect(httpClient *http.Client) error {
 	return nil
 }
 
+func (c *Client) metalApiConnect(httpClient *http.Client) error {
+	configuration := metal.NewConfiguration()
+	configuration.Debug = false
+	configuration.AddDefaultHeader("X-Auth-Token", c.Token())
```

Comment:
We should update the `configuration` to include `metal-cli` in the user-agent so that we maintain visibility into `metal-cli` usage:
```suggestion
	configuration.AddDefaultHeader("X-Auth-Token", c.Token())
	configuration.UserAgent = fmt.Sprintf(fmt.Sprintf("metal-cli/%s %s", c.Version, configuration.UserAgent)
```